lib/organization/Widgets/mydrawer.dart

```diff
@@ -0,0 +1,89 @@
+import 'package:flutter/material.dart';
+class mydrawer extends StatefulWidget {
+  const mydrawer({Key? key}) : super(key: key);
+
+  @override
+  State<mydrawer> createState() => _mydrawerState();
+}
+
+class _mydrawerState extends State<mydrawer> {
+  @override
+  Widget build(BuildContext context) {
+    return Drawer(
+    backgroundColor: Color.fromARGB(255, 166, 223, 250),
+    child: ListView(
```

Comment:
Use Column here in place of ListView. We don't want the profile_pic and main heading to be scrolled as well.